src/client/src/routes/IndexPage/components/IndexPage.js

```diff
@@ -8,7 +8,8 @@ import ProjectsDetails from './ProjectsDetails';
 import '../styles/IndexPage.css';
 
 const IndexPage = (props) => {
-  const { userId } = props;
+  // const { userId } = props;
+  const userId = '123456789';
   
```

Comment:
we need to create a component that holds all the forms information and also sets the userId in the sessionstorage that it recieves from the db. This will be implemented in the following sprint

Reply:
Ok, thanks for the explanation :)